PokeDemo/Controllers/HomeViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  ViewController.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 05/07/22.
+//
+// MARK: - Navigation
+
+import UIKit
+
+class HomeViewController: BaseViewController {
+    
+    // MARK: - IBOutlets
+    
+    @IBOutlet weak var pokemonTableView: UITableView!
+    
+    // MARK: - Properties
+    
+    // MARK: - Private
+
+    private let presenter = HomeViewPresenter(service: Service.shared)
+    private var pokemonList: [Pokemon]? = [Pokemon]()
+    
+    // MARK: - View Life Cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        pokemonTableView.register(UINib(nibName: PokemonTableViewCell.cellIdentifier(), bundle: nil), forCellReuseIdentifier: PokemonTableViewCell.cellIdentifier())
+        presenter.setViewDelegate(delegate: self)
+        presenter.getPokemonList()
+    }
+}
+
+// MARK: - Extensions
+
+extension HomeViewController: UITableViewDataSource, UITableViewDelegate {
```

Comment:
Divide the extension with separate protocol conformance. 

```
extension HomeViewController: UITableViewDataSource
```

```
HomeViewController: UITableViewDelegate
```

Reply:
Done in b692a34b7b2c496b64ffcb60893e180e0ad736ea